docs/index.adoc

```diff
@@ -102,6 +102,9 @@ For example, this concerns information how to build the final A2L file, if the F
 The `A2L` description depends on the FMU binary, for example, regarding memory addresses, and byte order.
 If an FMU archive contains multiple binaries for different platforms, the associated A2L files are placed into separate subfolders below `/extra/org.modelica.fmi.layered_XCP` following the same scheme as in the `/binaries` folder, see <<Structure of the FMU archive>>.
 
+Note, that these `A2L` files are not intended to be downloaded to the simulator platform.
+_[Any files that are required on the simulator shall be handled as defined by the FMI standard via the `/resource` folder.]_
+
```

Comment:
```suggestion
Note, that the `A2L` files placed under the `extra` directory are not accessible to the FMU at run time.
Any files that shall be accessible to the FMU at runtime must (also) be placed into the `resources` directory as defined by the FMI standard.

```

We should not talk about downloading, simulator platforms, etc., since they are not defined by the standards.

BTW, we should also likely change the wording to directories instead of folders across this document, since that is the wording that the FMI standard uses.